system/include/emscripten/val.h

```diff
@@ -368,6 +368,8 @@ class EMBIND_VISIBILITY_DEFAULT val {
     }
   }
 
+  val(val& v) : val(static_cast<const val&>(v)) {}
```

Comment:
Comment for why we have this would be helpful.

Reply:
Good idea, adding...